Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Controllers/MixedRealityControllerVisualizer.cs

```diff
@@ -1,113 +1,19 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
 using Microsoft.MixedReality.Toolkit.Internal.EventDatum.Input;
-using Microsoft.MixedReality.Toolkit.Internal.Extensions;
-using Microsoft.MixedReality.Toolkit.Internal.Interfaces;
-using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem.Handlers;
-using Microsoft.MixedReality.Toolkit.Internal.Managers;
-using Microsoft.MixedReality.Toolkit.Internal.Utilities;
-using Microsoft.MixedReality.Toolkit.SDK.Input;
+using Microsoft.MixedReality.Toolkit.SDK.Input.Handlers;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.SDK.UX.Controllers
 {
     /// <summary>
-    /// The Mixed Reality Visualization component is primarily responsible for rendering the user's controllers in a scene.
-    /// It will render either the specific model configured against the controller definition or fall back to using the generic/global default models defined in the Controllers Profile.
-    /// If no model found, then nothing is rendered, in which case the controllers will still operate as normal.
-    /// The visualizer can be enabled / disabled on demand once the project is started. If disabled on start, it cannot function.
+    /// The Mixed Reality Visualization component is primarily responsible for synchronizing the user's current input with controller models.
     /// </summary>
-    /// <example>
-    /// To Start, ensure there is a "Controllers Profile" configured against the main Mixed Reality Configuration Profile and that Controller Rendering is enabled in that profile, as well as configuring at least one controller for each platform you intend to support.
-    /// Once ready, then simply add this script to a GameObject in the scene, for example the Camera / head.
-    /// </example>
-    /// <remarks>
-    /// For Alpha, this visualizer only does basic rendering and management of controllers.  In future versions, it will expose specific transforms on a controller model (if it has them) to be able to attach items to any part of the model
-    /// It will also support animation of controller parts and possibly fading in and our of controllers.
-    /// </remarks>
     /// <seealso cref="Internal.Definitions.Devices.MixedRealityControllerMappingProfile"/>
-    public class MixedRealityControllerVisualizer : InputSystemGlobalListener, IMixedRealitySourcePoseHandler, IMixedRealityInputHandler
+    public class MixedRealityControllerVisualizer : ControllerPoseSynchronizer
```

Comment:
With the refactor, we can remove this class now as it doesn't do anything in ALpha

Reply:
I think it's okay to have a stub for future planning.

Besides I think David already decided this wasn't making it in alpha, but stabilization.

Reply:
Nothing is 100% decided.... let's at least get a beta task to do a cleanup pass